content/en/blog/falco-0-38-0/index.md

```diff
@@ -0,0 +1,153 @@
+
+This release brings an easier to use mechanism to install and configuring your drivers, new rule language features, better support for Falco metrics and many more improvements.
+
+During this release cycle, we merged more than 100 PRs on Falco and more than 180 PRs for libs and drivers, version 0.17.0 and version 7.1.1 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
+
+To learn all about these changes, read on!
+
+## What’s new? TL;DR
+
+*Key features:*
+
+* New capabilities in `falcoctl` to automatically select the best driver for your system
+* The Falco configuration file can now be split into multiple files to make it more manageable
+* Rule selection from configuration file or command line
+* Field transformers and value comparison
+* Prometheus metrics support
+* Plugin API enhancements
+
+This release also comes with [breaking changes](#breaking-changes) that you should be aware of before upgrading.
+
+## Major features and improvements
+
+The 0.38.0 release contains a number of feature and UX improvements, here are list of some of the key new capabilities.
+
+### Automatic driver selection ✨
+
+If we could pick the most common issue that we've heard from adopters and we experienced first hand is the fact that sometimes we all struggle with installing and upgrading Falco drivers. The Falco team has been tirelessly working for years to improve the installation experience and Linux kernel compatibility with massive changes such as the introduction of the new CO-RE eBPF probe and most recently the complete rewrite of our driver loading component. With this new version of `falcoctl`, integrated with Falco 0.38.0, our loading tool will automatically detect your system and pick the most compatible driver without any intervention; on recent kernel versions this is likely the modern eBPF probe. Of course, the tool also allows to select the preferred driver if the automatic choice is not optimal for your use case. Read more in the [installation](/docs/install-operate/installation/) documentation page.
+
+### Organize your Falco configuration files 🗃️
+
+Our `falco.yaml` configuration file gains more options, fine tuning configuration flags and feature selection for every release; in fact, they are so many that some people would like to better organize them in separate configuration files which can also be kept across Falco upgrades. Starting from this release you can add list of files or directory to the `config_files` configuration entry, which comes populated with the `/etc/falco/config.d/` directory by default. Read more in the [configuration options](/docs/reference/daemon/config-options/) section of the documentation.
```

Comment:
I'd mention that included config files are capable of overriding main config file configuration keys.

Reply:
Done!